### PR TITLE
library updates, whitespace, missing checksums, etc

### PR DESCRIPTION
**Pull request by totaam**
Hi John,

I've tried hard to keep this PR clean and separate the different sets of changes into many (too many?) separate commits.
Feel free to apply / cherry-pick / ignore at your own leisure.

Many thanks for your work on gtk-osx.
xpra on MacOS would not exist without you!

Edit: missed some more whitespace: c14125ec959d02a375e0e3d748d8699af5616018

### CODE REVIEW

File: modulesets-stable/gtk-osx.modules [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE moduleset SYSTEM "moduleset.dtd">
 <?xml-stylesheet type="text/xsl" href="moduleset.xsl"?>
 <moduleset>
-  <repository type="tarball" name="ftp.gnome.org" default="yes"
+  <repository type="tarball" name="download.gnome.org" default="yes"
               href="http://ftp.gnome.org/pub/GNOME/sources/"/>
```
Comment (jralls) [inline modulesets-stable/gtk-osx.modules:6]: You forgot to change the URI.

File: modulesets/gtk-osx-random.modules [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
    <cmake id="graphviz">
     <branch module="2.47.0/graphviz-2.47.0.tar.bz2" version="2.47.0"
-            repo="graphviz" 
+            repo="graphviz"
```
Comment (jralls) [inline modulesets/gtk-osx-random.modules:27]: Whitespace change in a non-whitespace commit.

File: modulesets/gtk-osx-random.modules [hunk 3]
```diff
@@ -35,7 +35,7 @@
   </cmake>
 
   <autotools id="vala" autogen-sh="configure">
-    <branch repo="ftp.gnome.org" version="0.52.0"
+    <branch repo="download.gnome.org" version="0.52.0"
```
Comment (jralls) [inline modulesets/gtk-osx-random.modules:38]: You made `download.gnome.org` the default repo so you can remove the repo attribute here like you did in modulesets-stable.

File: modulesets/gtk-osx-random.modules [hunk 4]
```diff
@@ -92,7 +92,7 @@
             module="pcre/pcre2-10.36.tar.bz2"
             hash="sha256:a9ef39278113542968c7c73a31cfcb81aca1faa64690f400b907e8ab6b4a665c"/>
   </autotools>
-  
+
```
Comment (jralls) [inline modulesets/gtk-osx-random.modules:95]: Whitespace in a non-whitespace commit.

File: modulesets/gtk-osx-random.modules [hunk 5]
```diff
@@ -101,7 +101,7 @@
     </dependencies>
   </meson>
 
-<autotools id="getopt" autogen-sh="configure"
+  <autotools id="getopt" autogen-sh="configure"
```
Comment (jralls) [inline modulesets/gtk-osx-random.modules:104]: Whitespace in a non-whitespace commit.

File: modulesets/gtk-osx-random.modules [hunk 6]
```diff
@@ -171,7 +171,6 @@
 
   <autotools id="enchant" autogenargs="--enable-relocatable"
              autogen-template="pushd %(srcdir)s &amp;&amp; ./%(autogen-sh)s &amp;&amp; popd &amp;&amp; %(srcdir)s/configure --prefix=%(prefix)s %(autogenargs)s">
-
```
Comment (jralls) [inline modulesets/gtk-osx-random.modules:174]: Whitespace in a non-whitespace commit.

File: modulesets/gtk-osx-random.modules [hunk 7]
```diff
@@ -210,5 +209,5 @@
     </branch>
   </autotools>
 
- 
+
```
Comment (jralls) [inline modulesets/gtk-osx-random.modules:212]: Whitespace in a non-whitespace commit.

**Review by jralls — CHANGES_REQUESTED** [review]
Thanks. Please rebase to put all of the whitespace changes in a single commit.

**Comment by totaam** [comment]
Replacing by:
https://github.com/jralls/gtk-osx-build/pull/78

**Comment by jralls** [comment]
> Replacing by:

You don't have to do that. Just force-push the rebased branch and github/gitlab will do the right thing wth the PR.

**Comment by totaam** [comment]
> Just force-push the rebased branch and github/gitlab will do the right thing wth the PR

Ah! Didn't know that.
Anyway, I'm not sure I would get the rebasing done correctly and this was easy.

File: modulesets/gtk-osx-random.modules (hunk 3 above)
Comment (totaam) [reply modulesets/gtk-osx-random.modules:38]: Oops. That was a mistake!
`modulesets-stable` can switch the default safely but `modulesets` cannot: it uses `git.gnome.org` as default, not 'download.gnome.org'.

**Pull request closed without merging**